config/spryk/spryks/Spryker/Glue/ApiApplication/Plugin/AddGlueApiApplicationResourceMethod.yml

```diff
@@ -43,4 +43,4 @@ arguments:
         description: "Can be oneOf: Get, GetCollection, Post, Patch, Delete, DeleteCollection"
```

Comment:
Is there a possibility to make this a choice? // Please only use existing functionality, no need to invest too much time into this.

Reply:
We pass it through the `--httpMethod=get` option so we can only manipulate with the prefix as for now.